lib/browser/chrome-extension.js

```diff
@@ -426,4 +419,21 @@ app.once('ready', function () {
     })
     return devExtensions
   }
+
+  // Load persisted extensions.
+  loadedDevToolsExtensionsPath = path.join(app.getPath('userData'), 'DevTools Extensions')
```

Comment:
Looks like this goes into the global pool. Maybe a `const`?

Reply:
I didn't write the original code, but it looks like the reasoning is that this path is used in both `app.once('ready')` and `app.on('will-quit')`. I'm guessing that it's not being assigned a value until the app is ready because `app.getPath('userData')` won't be guaranteed until the app ready event is called. (based on this documentation for setPath [https://github.com/electron/electron/blob/master/docs/api/app.md#appsetpathname-path](url))

So it looks like we need to keep it as a `let` so that the re-assignment can happen.